teamworkApp/app/views/student_graph/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Teamwork Analysis</title>
+  <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track" => true %>
+  <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+    <div class="container" style="padding:70px">
+        <div class="row">
+            <div class="col-lg-3">
+                <center>
+                    <h4>Students</h4>
+                </center>
+                <%= simple_form_for @selection do |f| %>
+                    <% @students.each do |student| %>
+                        <%= f.input :selected, as: :check_boxes, collection: [student], :input_html => { :checked => @selected[student.id.to_s] }, label: false %>
+                     <% end %>
```

Comment:
This works now if the student ids don't start at 1 right? because if data is deleted the new data inputted will start after the last id number allocated.

Reply:
Yes, it works now!